pipeline/rating/tasks/task-helm-lint.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: helm-lint
+spec:
+  params:
+  - name: url
+    type: string
+  - name: component
+    type: string
+  - name: version
+    type: string
+  results:
+  - name: LINT
+    type: string
+  steps:
+    - name: helm-lint
+      image: kubebb/component-rbac-lint:v0.0.1
```

Comment:
Since there would be many task besides `rback` and `lint`, `kubebb/component-rating` would be much better